viadot/tasks/mindful.py

```diff
@@ -31,18 +33,38 @@ def __init__(
             end_date (datetime, optional): End date of the resquest. Defaults to None.
             date_interval (int, optional): How many days are included in the request.
                 If end_date is passed as an argument, date_interval will be invalidated. Defaults to 1.
+            file_extension (Literal[parquet, csv], optional): file extensions for storing responses. Defaults to "csv".
+            file_path (str, optional): Path where to save the file locally. Defaults to ''.
         """
         self.credentials_mindful = credentials_mindful
         self.start_date = start_date
         self.end_date = end_date
         self.date_interval = date_interval
+        self.file_extension = file_extension
+        self.file_path = file_path
 
         super().__init__(
             name=report_name,
             *args,
             **kwargs,
         )
 
+        if not isinstance(start_date, datetime):
+            self.start_date = datetime.now()
+            self.end_date = self.start_date + timedelta(days=date_interval)
```

Comment:
Is this how it works? : 
`start_date = today, end_date=1 (as default) `
so the range will be from `today->tomorrow`? Do we extract data from the future? (If the data interval is >1 and start_date =today)